lib/dialects/abstract/query-generator.js

```diff
@@ -1612,7 +1607,6 @@ var QueryGenerator = {
     }
 
     query += ';';
-
```

Comment:
Please remove the whitespace changes (it makes the changes easier to cherry-pick onto master)